### PR TITLE
Cleanup requirements

### PR DESCRIPTION
**Pull request by Leppunen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,4 @@ userdispatch.py
 venv/
 
 .idea/
+.vscode/
```
Comment (RAnders00) [inline .gitignore]: LF at the end of the file missing here

File: install/venvinstall.sh [hunk 2]
```diff
@@ -8,14 +8,24 @@ if [ ! -d install ]; then
...
 
 # Upgrade pip
-pip install pip --upgrade
+./venv/bin/pip install pip --upgrade
 
 # Install requirements.txt
-pip install -r requirements.txt
+./venv/bin/pip install -r requirements.txt
+
+if [ $CIRCLECI ]; then
```
Comment (RAnders00) [inline install/venvinstall.sh]: should be `$CI` to make it more portable

File: requirements-dev.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+pluggy==0.12.0
+py==1.8.0
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pyparsing==2.4.2
+pytest==5.1.2
+six==1.12.0
+toml==0.10.0
+wcwidth==0.1.7
+zipp==0.6.0
```
Comment (RAnders00) [inline requirements-dev.txt]: I think `pytest` should be in `requirements-dev.txt`, and CircleCI should use `venvinstall.sh --dev`. Then you wouldn't have to detect `$CI` in `venvinstall.sh`.

Comment (Leppunen) [reply requirements-dev.txt]: Cannot be done until Python 3.5 is removed from CircleCI as it throws an error while trying to install black which is in the dev requirements.

File: install/venvinstall.sh (hunk 2 above)
Comment (Leppunen) [reply install/venvinstall.sh]: This is now fixed.

File: .gitignore (hunk 1 above)
Comment (Leppunen) [reply .gitignore]: This is now fixed.

**Pull request merged**